extensions/spark/kyuubi-spark-connector-hive/src/main/spark-3.4/org/apache/kyuubi/spark/connector/hive/PartitionedFileUtils.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.spark.connector.hive
+
+import org.apache.spark.sql.execution.datasources.PartitionedFile
+
+object PartitionedFileUtils {
```

Comment:
we don't need to introduce spark-3.3 and spark-3.4 folders, please use reflection to resolve it.

BTW, it's expected that the connector compiled against spark-3.3 can work on both 3.3 and 3.4

Reply:
Thanks, updated